docs/manage/fleet/system-settings.md

```diff
@@ -0,0 +1,199 @@
+---
+linkTitle: "Configure machine settings"
+title: "Configure machine operating system settings"
+weight: 50
+layout: "docs"
+type: "docs"
+description: "Configure network settings, operating system package updates and logging defaults."
+---
+
+The `viam-agent` configuration allows you to configure:
+
+- [settings for package updates for the host operating system](#configure-os-package-updates)
+- [networks a machine can connect to](#configure-networks)
+- [parameters for operating system logging](#configure-operating-system-logging)
+
+## Configure OS package updates
+
+By default, the configuration in <FILE>/etc/apt/apt.conf.d/</FILE> determines the behavior for updating operating system packages.
+To adjust these settings update the `"agent"` value in the machine's JSON configuration.
+
+For complete reference information, see [viam-agent](/configure/agent/#agent-syscfg).
+
+### Automatically upgrade all packages
+
+When set, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
+Custom repos installed on the system at the time the setting is enabled will be included.
+
+To set automatic upgrades for all packages, add the `upgrades` field to the `agent-syscfg`'s `attributes` object as following.
+You may need to add the `agent-syscfg` object to the `agent` object if it doesn't already exist.
+
+
+```json
+"agent": {
+    "agent-syscfg": {
+        "release_channel": "stable",
+        "attributes": {
+            "upgrades": {
+                "type": "all"
+            }
+        }
+    }
+}
+```
+
+### Automatically upgrade security packages
+
+When set, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
+Custom repos installed on the system at the time the setting is enabled will be included.
+
+To set automatic upgrades for packages containing `"security"` in their codename (for example `bookworm-security`), add the `upgrades` field to the `agent-syscfg`'s `attributes` object as following.
+You may need to add the `agent-syscfg` object to the `agent` object if it doesn't already exist.
```

Comment:
Why is this explanation repeated for every option here? Wouldn't it make more sense to just explain the overall purpose once at the top?